testcontainers/mssql.py

```diff
@@ -25,14 +25,15 @@ class SqlServerContainer(DbContainer):
     """
 
     def __init__(self, image="mcr.microsoft.com/mssql/server:2019-latest", user="SA", password=None,
-                 port=1433, dbname="tempdb", dialect='mssql+pymssql', **kwargs):
+                 port=1433, dbname="tempdb", dialect: str = 'mssql+pyodbc', driver: str = "ODBC Driver 17 for SQL Server", **kwargs):
```

Comment:
I don't like setting the standard driver here to a versioned driver which will be deprecated at some point (18 is already out). And the driver is an issue with defaulting to ODBC. What I would do is leave the default as pymssql, set the default driver to None, but throw an error if someone asks for mssql+pyodbc with no driver. Then in our code, pass in mssql+pyodbc.

The reason to do it this way is this is not our library, we would like to keep our additions and modifications in such a state that they can choose to integrate them if they wish, and if I owned the library, I would not merge this change the way it currently is.

Reply:
Makes perfect sense.